storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -1309,32 +1309,50 @@ function innodb_checkpoint_lsn()
         grep "Last checkpoint at" | awk '{ print $4 }'
 }
 
+#method to check count in a database/table
+#@in database
+#@in table
+#@in expected row count
+########################################################################
+function check_count() {
+  local db=$1
+  local table=$2
+  local expected_row_count=$3
+  local row_count=`$MYSQL $MYSQL_ARGS -Ns -e "select count(1) from $table" $db | awk {'print $1'}`
+  vlog "row count in table $table is $row_count"
+
+  if [ "$row_count" != $expected_row_count ]; then
+   vlog "rows in table t1 is $row_count when it should be $expected_row_count"
+   exit -1
+  fi
+}
+
 #general method to restore
 ########################################################################
 function backup_and_restore() {
-  local db=$1;
+  local db=$1
   record_db_state $db
   rm -rf $topdir/backup
   mkdir -p $topdir/backup
 
   ## backup and prepare based on if keyring is used
   if [ -z ${plugin_dir+x} ]
   then
-    run_cmd xtrabackup --backup --target-dir=$topdir/backup
+    xtrabackup --backup --target-dir=$topdir/backup
```

Comment:
why run_cmd is removed?

Reply:
xtrabackup function internally call run_cmd that's why we don't require explicitly.

Reply:
'xtrabackup' here is the binary right? not the function? :thinking:

Reply:
It is function defined in commoh.sh